metrics/METRICS.md

```diff
@@ -1,5 +1,18 @@
 This section describes how it's possible to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The provided resources for deploying can't be considered production grade; they are used just as an example to show how it's possible to use the metrics feature provided in this project.
 
+If you are going to use `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default number of CPU cores and memory used by the related virtual machine (i.e. not using the default 2 cores and 2 GB of RAM but 4 cpus and 4 GB  of RAM instead).
```

Comment:
Do we really need the CPUs? Shouldn't we need only the memory?

Reply:
You are right, I tested without increasing cpus and it seems to work fine.